tests/installation/agama_reboot.pm

```diff
@@ -38,9 +42,63 @@ sub get_agama_install_console_tty {
     return 7;
 }
 
+sub verify_agama_auto_install_done_cmdline {
+    # for some remote workers, there is no vnc access to the install console,
+    # so we need to make sure the installation has completed from command line.
+
+    # live root can support ssh access
+    my $timeout = 200;
+    while ($timeout > 0) {
+        if (script_run("journalctl -u agama | grep 'Install phase done'") == 0) {
+            record_info("agama install phase done");
+            return;
+        }
+        sleep 20;
+        $timeout = $timeout - 20;
+    }
+    die "Install phase is not done, please check agama logs";
+}
+
+sub agama_system_prepare_ipmi {
+    # Mount the root disk after installation, and do some prepare tasks:
+    # Configure serial console, enable root ssh access, etc
+
+    my $device = get_var("AGAMA_ROOT_TARGET_DEVICE", "/dev/sda");
```

Comment:
same comment than before, if you can guess this in a different way or if you can avoid `AGAMA_*` variables would be better, anyway this is already the target system.

Reply:
Hmm, Let me try to use agama config command to show the root disk name